backend/package.json

```diff
@@ -61,7 +61,7 @@
     "graphql-custom-directives": "~0.2.14",
     "graphql-iso-date": "~3.6.1",
     "graphql-middleware": "~3.0.2",
-    "graphql-shield": "~5.7.1",
+    "graphql-shield": "~5.6.1",
```

Comment:
downgraded because `5.7.1` was breaking the `CreatePost` implementation

Reply:
for all the lovely `PR` reviewers out there, there is this issue https://github.com/maticzav/graphql-shield/issues/408, which talks about what could be happening, and possible solutions